src/rdkafka_conf.c

```diff
@@ -3248,6 +3248,7 @@ void rd_kafka_conf_properties_show (FILE *fp) {
 		case _RK_C_STR:
                 case _RK_C_KSTR:
 			typeinfo = "string";
+			break;
```

Comment:
It should fall through (eventually hitting line 3262 where it prints prop->sdef, if anything there should be a `/*FALLTHRU*/` comment here.